tox.ini

```diff
@@ -57,7 +57,7 @@ commands = pytest -v --basetemp=_pytest_temp -s -m "todo" deluge/tests
 setenv = {[testenv]setenv}:{toxinidir}/deluge/tests
 commands =
     python -c "import libtorrent as lt; print lt.__version__"
-    python -m twisted.trial --reporter=deluge-reporter deluge.tests
+    python -m twisted.trial deluge.tests
```

Comment:
Why is the deluge-reporter being removed?

Reply:
It stops the test from being run at all. The setup goes fine but then at the first command fails. says something about only passing report to something...I don't remember at the moment.